aspnetcore/razor-pages/rp-mvc.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Razor Pages compared to ASP.NET Core MVC with controllers and views
+author: Rick-Anderson
+description: Contrast and compare Razor Pages to controller/view programming in ASP.NET Core
+monikerRange: '>= aspnetcore-2.0'
+ms.author: riande
+ms.date: 8/05/2018
+uid: razor-pages/rp-mvc
+---
+# Razor Pages compared to ASP.NET Core MVC
+
+By [Scott Sauber](https://twitter.com/scottsauber) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Razor Pages is a feature of the ASP.NET Core MVC framework. Razor Pages follows the MVVM pattern. Developers and teams familiar with ASP.NET MVC development:
+
+* Can continue app development featuring controllers and views.
+* Can be assured the ASP.NET Core framework will provide improvements to controller/view development.
+
+Razor Pages can be added to controller/view projects. The controller-view approach to MVC may be preferred by larger teams.
+
+If you haven't tried Razor Pages, consider using them. Many developers find Razor Pages development simpler and more productive than using controllers and views.
+
+Razor Pages advantages over controller/view development:
+
+* **A better folder structure by default that scales better.**  In MVC, the default folder structure  does not scale.  Separate folders for Views, Controllers, and often ViewModels when all three are tightly coupled to one another.  You must bounce between all three folders anytime you need to add or change a feature.  With Razor Pages, your `PageModel` (Controller + ViewModel) are in the same folder as your View.  You hit F7 in Visual Studio to toggle between them.
```

Comment:
PageModel are --> PageModel is
does not scale --> is not easy to work with
(what you're describing isn't so much scaling as a structure that is intrinsically awkward to work with)